File: files/FOANUR.java
import java.util.Scanner;
class fact (int);
public class Main
{
    public static void main(String[] args) {
        Scanner uwu = new Scanner (System.in);
        int num, f;
        num = uwu.nextInt ();
        f = fact (num);
    }
}
class fact (int aj) {
    if (aj == 1 || aj == 0)
        return 1;
    else 
        return (aj * fact (aj - 1));
}

File: files/FOANUR.c
#include<stdio.h>
int fact(int);
int main()
{
    int num, f;
    scanf("%d", &num);
    f= fact(num);
    return 0;
}

int fact(int aj)
{
    if(aj==1 || aj==0)
        return 1;
    else
        return (aj*fact(aj-1));
}